testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/ClientAuthSignedJWTTest.java

```diff
@@ -845,7 +845,9 @@ protected JsonWebToken createRequestToken(String clientId, String realmInfoUrl)
 
             int now = Time.currentTime();
             if (isClaimEnabled("issuedAt")) reqToken.issuedAt(now);
-            if (isClaimEnabled("expiration")) reqToken.expiration(now + getTokenTimeout());
+            // For the time being there's no getter for tokenTimeout in JWTClientCredentialsProvider
```

Comment:
That crossed my mind too, but it didn't seem to me like very "objective" solution - it would break some object design principles (hiding a property). Adding a getter (either public or protected) looked like much more cleaner solution.